tests/unit/spec/base/core-datalayer.js

```diff
@@ -25,6 +25,60 @@ describe('core-datalayer.js', function() {
         });
     });
 
+    describe('updateDataLayerPush', function() {
+
+        beforeAll(function() {
+            Mozilla.Analytics.updateDataLayerPush();
+        });
+
+        beforeEach(function() {
+           var link = '<a id="link" href="https://www.mozilla.org/en-US/firefox/new/">';
+            $(link).appendTo('body');
+            window.linkElement = $('#link')[0];
```

Comment:
What I mean is put the variable inside the function scope of `updateDataLayerPush`:

```javascript
describe('updateDataLayerPush', function() {

    var linkElement;

    beforeEach(function() {
        var link = '<a id="link" href="https://www.mozilla.org/en-US/firefox/new/">';
        $(link).appendTo('body');
        linkElement = $('#link')[0];
    });
```